pkg/tfbridge/provider_test.go

```diff
@@ -5623,3 +5623,97 @@ func TestPlanResourceChangeUnknowns(t *testing.T) {
 	}`)
 	})
 }
+
+func TestSetDuplicatedDiffEntries(t *testing.T) {
```

Comment:
Lets add a note here explaining why it's bad to send duplicate entries to the bridge. Ideally we would reference the issue we are addressing and the issue in pulumi/pulumi that details why they can't handle duplicate entries.